docs/user-guide/ze-profiles.md

```diff
@@ -1,22 +1,18 @@
-# Zowe Explorer profiles
+# Creating Zowe Explorer profiles
 
 After you install Zowe Explorer, you must have a Zowe Explorer profile to use all functions of the extension.
-
-:::info
-You can continue using Zowe V1 profiles with Zowe Explorer V2. See [Working with Zowe CLI V1 profiles](#working-with-zowe-cli-v1-profiles) for more information.
+:::info Required roles: Systems administrator
 :::
 
-## Configuring Zowe V2 profiles
-
-Zowe V2 uses *team profiles* to simplify profile management by letting you edit, store, and share mainframe connection details in one location, a configuration file.
+## Configuring Zowe profiles
 
-You can use a text editor or an IDE to populate configuration files with the connection information for your mainframe services. By default, your *global* team configuration file is located in the `.zowe` home folder, whereas the *project* configuration file is located in the main directory of your project.
+Zowe uses [team configuration](/appendix/zowe-glossary.md#team-configuration) to simplify profile management by letting you edit, store, and share mainframe connection details in one location, a configuration file.
 
-You can create profiles that you use globally, given that the names of the globally-used profiles are different from your other profile names.
+You can use a text editor or an IDE to populate configuration files with profiles, which contain the connection information for your mainframe services. By default, your *global* team configuration file is located in the `.zowe` home folder, whereas a *project* configuration file is located in the main directory of your project.
 
 :::note
 
-When multiple profiles are available in Zowe CLI, project configuration takes precedence over global configuration. To learn more, see [How Zowe CLI uses configurations](../user-guide/cli-using-understand-profiles-configs).
+When multiple profiles with the same name are available in Zowe CLI **[is it accurate to say zowe cli here?]**, project configuration takes precedence over global configuration. To learn more, see [How Zowe CLI uses configurations](../user-guide/cli-using-understand-profiles-configs).
```

Comment:
@adam-wolfe please respond

Reply:
I would say Zowe Explorer here. It's true for both Zowe CLI and Explorer, but in the context of this article, it would be better to say Zowe Explorer.